internal/services/gateway/action/org.go

```diff
@@ -131,6 +133,17 @@ func (h *ActionHandler) DeleteOrg(ctx context.Context, orgRef string) error {
 		return util.NewAPIError(util.ErrForbidden, errors.Errorf("user not authorized"))
 	}
 
+	orgInvitations, _, err := h.configstoreClient.GetOrgInvitations(ctx, orgRef, 0)
+	if err != nil {
+		return util.NewAPIError(util.KindFromRemoteError(err), err)
+	}
+	for _, invitation := range orgInvitations {
+		_, err = h.configstoreClient.DeleteInvitation(ctx, orgRef, invitation.UserID)
+		if err != nil {
+			return util.NewAPIError(util.KindFromRemoteError(err), err)
+		}
+	}
+
```

Comment:
This should be done in the configstore that is responsible for maintaining data consistency